nixos/tests/all-tests.nix

```diff
@@ -969,6 +969,7 @@ in {
   stalwart-mail = handleTest ./stalwart-mail.nix {};
   stargazer = runTest ./web-servers/stargazer.nix;
   starship = handleTest ./starship.nix {};
+  stash = handleTest ./stash.nix {};
```

Comment:
should be limited to aarch64-linux and x86_64-linux only.

`handleTestOn ["aarch64-linux" "x86_64-linux"] ./stash.nix {};`